src/libexpr/print-options.hh

```diff
@@ -49,4 +49,18 @@ struct PrintOptions
     size_t maxStringLength = std::numeric_limits<size_t>::max();
 };
 
+/**
+ * `PrintOptions` suitable for debugging.
+ *
+ * These options are used for printing values in error messages without
+ * printing "too much" output.
```

Comment:
Debugging is a bit vague. We're not using it in the (repl) debugger.

```suggestion
 * `PrintOptions` for unknown and therefore potentially large values in error messages,
 * to avoid printing "too much" output.
```